code/modules/projectiles/ammunition/_ammunition.dm

```diff
@@ -67,17 +67,34 @@
 /obj/item/ammo_casing/proc/add_notes_ammo()
 	// Try to get a projectile to derive stats from
 	var/obj/projectile/exam_proj = projectile_type
+	// projectile damage multiplier for guns with snowflaked damage multipliers
+	var/proj_damage_mult = 1
 	if(!ispath(exam_proj) || pellets == 0)
 		return
 
+	// are we in an ammo box?
+	if(isammobox(loc))
+		var/obj/item/ammo_box/our_box = loc
+		// is our ammo box in a gun?
+		if(isgun(our_box.loc))
+			var/obj/item/gun/our_gun = our_box.loc
+			// grab the damage multiplier
+			proj_damage_mult = our_gun.projectile_damage_multiplier
+	// if not, are we just in a gun e.g. chambered
+	else if(isgun(loc))
+		var/obj/item/gun/our_gun = loc
+		// grab the damage multiplier.
+		proj_damage_mult = our_gun.projectile_damage_multiplier
+
 	var/list/readout = list()
 	// No dividing by 0
-	if(initial(exam_proj.damage) > 0)
-		readout += "Most monkeys our legal team subjected to these [span_warning(caliber)] rounds succumbed to their wounds after [span_warning("[HITS_TO_CRIT(initial(exam_proj.damage) * pellets)] shot\s")] at point-blank, taking [span_warning("[pellets] shot\s")] per round"
-	if(initial(exam_proj.stamina) > 0)
-		readout += "[!readout.len ? "Most monkeys" : "More fortunate monkeys"] collapsed from exhaustion after [span_warning("[HITS_TO_CRIT(initial(exam_proj.stamina) * pellets)] impact\s")] of these [span_warning("[caliber]")] rounds"
+	if(proj_damage_mult)
+		if(initial(exam_proj.damage) > 0)
+			readout += "Most monkeys our legal team subjected to these [span_warning(caliber)] rounds succumbed to their wounds after [span_warning("[HITS_TO_CRIT((initial(exam_proj.damage) * proj_damage_mult) * pellets)] shot\s")] at point-blank, taking [span_warning("[pellets] shot\s")] per round."
+		if(initial(exam_proj.stamina) > 0)
+			readout += "[!readout.len ? "Most monkeys" : "More fortunate monkeys"] collapsed from exhaustion after [span_warning("[HITS_TO_CRIT((initial(exam_proj.stamina) * proj_damage_mult) * pellets)] impact\s")] of these [span_warning("[caliber]")] rounds."
 	if(!readout.len) // Everything else failed, give generic text
-		return "Our legal team has determined the offensive nature of these [span_warning(caliber)] rounds to be esoteric"
+		return "Our legal team has determined the offensive nature of these [span_warning(caliber)] rounds to be esoteric."
```

Comment:
you should return this earlier if the project has zero or less damage and stam damage, or if the projectile multiplier if zero or less.